### PR TITLE
translation: `hi`  translations for all the DOM reference

### PR DESCRIPTION
**Pull request by EshaanAgg**

### CODE REVIEW

File: src/content/reference/hi/p5/createFileInput.mdx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+submodule: डोम
+file: src/dom/dom.js
+description: >
+  <p>एक <code>'फ़ाइल'</code> <code><input></input></code> प्रकार का तत्व बनाता है | </p>
+
+  <p><code>createFileInput()</code> उपयोगकर्ताओं को स्केच में उपयोग के लिए स्थानीय फ़ाइलों का चयन करने की अनुमति देता है। यह एक <a href="/reference/p5/p5.File">p5.File</a> ऑब्जेक्ट लौटाता है।</p>
+
+  <p>पहला पैरामीटर, <code>callback</code>, एक फ़ंक्शन है जिसे फ़ाइल लोड होने पर कॉल किया जाता है। callback फ़ंक्शन में एक पैरामीटर होना चाहिए, <code>file</code>, जो एक <a href="/reference/p5/p5.File">p5.File</a> ऑब्जेक्ट है।</p>
+
+  <p>दूसरा पैरामीटर, <code>एकाधिक</code>, वैकल्पिक है। यह एक बूलियन मान है जो <code>true</code> पर सेट होने पर एकाधिक फ़ाइलें लोड करने की अनुमति देता है। यदि <code>true</code> है, तो <code>callback</code> को प्रति फ़ाइल एक बार कॉल किया जाएगा।</p>
```
Comment (TanviKumar) [inline src/content/reference/hi/p5/createFileInput.mdx:13]: Code should not be translated

**Review by TanviKumar — CHANGES_REQUESTED** [review]

**Comment by TanviKumar** [comment]
@EshaanAgg PR looks good, just one minor requested change. Kindly make that, will approve once done

File: src/content/reference/hi/p5/createFileInput.mdx (hunk 1 above)
Comment (EshaanAgg) [reply src/content/reference/hi/p5/createFileInput.mdx:13]: Nice catch! Will update.

**Comment by EshaanAgg** [comment]
Thanks for the review @TanviKumar! I have made the required changes.

**Review by TanviKumar — APPROVED** [review]

**Pull request merged**